### ISSUE TITLE
[Feature Request]Can we add mermaid as diagram library?

### ISSUE THREAD

**Issue by zxdposter** [HappenApps/Quiver#889]
[mermaid](http://knsv.github.io/mermaid/#mermaid)

This is a good diagram library, has flowchart sequence grantt.

I think quiver well be more powerful with this.

like this

![test](https://cloud.githubusercontent.com/assets/13846260/26767926/bdaedfbe-49d6-11e7-85cc-b22b4a5949b4.png)

![test](https://cloud.githubusercontent.com/assets/13846260/26767983/285e403e-49d7-11e7-8c56-95a4a261963c.png)

![test](https://cloud.githubusercontent.com/assets/13846260/26768004/4642901e-49d7-11e7-8a02-f50c109653e0.png)

![test](https://cloud.githubusercontent.com/assets/13846260/26768029/6d28ead4-49d7-11e7-8d63-d26ed391e775.png)

**Comment by egel** [comment]
Yes, I completely agree with @zxdposter. The notebook will really benefit from implementing this library.

**Comment by jeeftor** [comment]
I fully support this!!!

**Comment by hu-chia** [comment]
agree and expect
